src/Security/MemberAuthenticator/CookieAuthenticationHandler.php

```diff
@@ -39,70 +42,77 @@ class CookieAuthenticationHandler implements AuthenticationHandler
 
     /**
      * Get the name of the cookie used to track this device
-     *
-     * @return string
      */
-    public function getDeviceCookieName()
+    public function getDeviceCookieName(): string
     {
-        return $this->deviceCookieName;
+        return $this->deviceCookieName ?? '';
     }
 
     /**
      * Set the name of the cookie used to track this device
-     *
-     * @param string $deviceCookieName
-     * @return $this
      */
-    public function setDeviceCookieName($deviceCookieName)
+    public function setDeviceCookieName(string $deviceCookieName): static
     {
         $this->deviceCookieName = $deviceCookieName;
         return $this;
     }
 
     /**
      * Get the name of the cookie used to store an login token
-     *
-     * @return string
      */
-    public function getTokenCookieName()
+    public function getTokenCookieName(): string
     {
-        return $this->tokenCookieName;
+        return $this->tokenCookieName ?? '';
```

Comment:
```suggestion
        return $this->tokenCookieName;
```
Same as getDeviceCookieName() above